service/templates/base-auth.html

```diff
@@ -3,7 +3,7 @@
 {% block assets %}
 {{ super() }}
 
-{% with core_styles_cdn_url="https://cdn.jsdelivr.net/npm/@tacc/core-styles@v2.30.0" %}
+{% with core_styles_cdn_url="https://cdn.jsdelivr.net/npm/@tacc/core-styles@v2.30.1" %}
```

Comment:
**What?** Allow a busy button to act disabled even if it is not `disabled`. (The final "Connect" button must not be disabled.)